fe/src/components/campaign/fund-form.vue

```diff
@@ -24,18 +24,35 @@
           Supporting a project for no reward, just because it speaks to you.
         </p>
       </div>
-      <button type="submit" class="btn-gradient-hr">Fund Campaign</button>
+      <BaseButton
+        type="submit"
+        :class="
+          isLoading
+            ? 'bg-disabled h-9 px-4 rounded-[6px] text-white'
+            : 'btn-gradient-hr'
+        "
```

Comment:
Can we add this logic of isLoading class to the basebutton component instead? 
Then pass `isLoading` props